Update iam.mdx (#3448)
hashicorp/boundary: website/content/docs/concepts/iam.mdx

[permissions](/boundary/docs/concepts/security/permissions) which are granted to any principal (user or group) from any scope. Roles
can be contained by any scope, and the permissions can be applied to the same scope or any child scope.

> <Tip>

Author: @Dan-Heath would it be appropriate to change this into some kind of note saying that we want to change this to support multiple roles in the future? This is a minor change we publicly comment on, not sure if it is acceptable in documentation

Reviewer: I'd recommend against describing future plans like this in the docs. It could potentially be confusing to describe the way we want a feature to work in the future. Readers might not remember the future caveat later and mistakenly think it is supported now. Also, the text itself can become dated quickly if the update doesn't happen according to our anticipated schedule.

Author: Makes sense, won't put that in

3. In the same scope, create a new **user**, and then attach the new account to the new user. The user is now able to authenticate to Boundary.

### Configure users - OIDC/LDAP

> With the [OIDC](/boundary/tutorials/identity-management/oidc-auth) and LDAP auth methods, Boundary automatically generates accounts and users in the same scope
> as the auth method, using data provided by the identity provider. The accounts and users are only created once the user authenticates to Boundary for the first time. 

Reviewer: ```suggestion
With the [OIDC](/boundary/tutorials/identity-management/oidc-auth) and LDAP auth methods, Boundary uses data from the identity provider to automatically generate accounts and users in the same scope
as the auth method. The accounts and users are only created once the user authenticates to Boundary for the first time. 
```
